data/application.css

```diff
@@ -43,6 +43,13 @@ toolbox.dark {
     background-color: #33393b;  /* Adwaita-dark background color */
 }
 
+/* Toolbox HeaderBar */
+frame.toolbox > border,
+frame.toolbox > toolbox,
+frame.toolbox > toolbox > headerbar {
```

Comment:
Do you need all three selectors? Why isn't the headerbar only not enough?

Reply:
because the parent containers also draw backgrounds or borders so you need all three of them